srcpkgs/systemd-boot/files/kernel.d/systemd-boot.post-install

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+#
+# Kernel hook for systemd-boot.
+#
+# Arguments passed to this script: $1 pkgname, $2 version.
+#
+PKGNAME="$1"
+VERSION="$2"
+
+. "$ROOTDIR/etc/default/systemd-boot"
+
+if [ "$SYSTEMD_BOOT_DISABLE" ]; then
+	exit 0
+fi
+
+is_installed="$(bootctl is-installed)"
+
+if [ "$is_installed" != "yes" ]; then
+	echo "systemd-boot is not installed to the ESP"
+	echo "run 'bootctl install' to install it"
+	exit 0
+fi
```

Comment:
```suggestion
if ! bootctl is-installed >/dev/null; then
	echo "systemd-boot is not installed to the ESP"
	echo "run 'bootctl install' to install it"
	exit 0
fi
```
It seems that [`bootctl` returns error when this query is negative](https://github.com/systemd/systemd/blob/main/src/boot/bootctl-install.c#L1135-L1143), so avoiding output scraping is preferable here.